build.sbt

```diff
@@ -10,40 +10,26 @@ packageDescription := "Guardian front pages editor"
 
 ThisBuild / scalaVersion := "2.13.5"
 
-import com.gu.riffraff.artifact.BuildInfo
 import sbt.Resolver
 
-debianPackageDependencies := Seq("openjdk-8-jre-headless")
+debianPackageDependencies := Seq("java11-runtime-headless")
```

Comment:
Just FMI as I'm sure it is – why is this necessary? Doesn't the AMI provide a Java runtime?

Reply:
_Technically_ it's not necessary - but it's good practise when building a java-based DEB to have a generic JRE dependency

Reply:
Andy is correct that technically it isn't necessary. But if this line is present, it needs to be correct. AFAIK if you left this as openjdk-8-jre-headless the package would fail to install.